Makefile

```diff
@@ -20,6 +20,7 @@ install_env:
 		source $(CONDA_PREFIX)/etc/profile.d/conda.sh && \
 		conda create -n $(ENV_NAME) -y -c conda-forge cyipopt python=3.7 && \
 		conda activate $(ENV_NAME) && \
+		conda install --yes h5py && \
```

Comment:
Does this grab the clib and the python bindings?

Reply:
If i spin up a new anaconda env and then *only* install h5py here's the list of installed packages

```
$ conda install --yes h5py
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /ihme/homes/miker985/anaconda3/envs/test-h5py-install

  added / updated specs:
    - h5py


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    blas-1.0                   |              mkl           6 KB
    h5py-2.10.0                |   py36hd6299e0_1         901 KB
    intel-openmp-2020.1        |              217         780 KB
    mkl-2020.1                 |              217       129.0 MB
    mkl-service-2.3.0          |   py36he904b0f_0         219 KB
    mkl_fft-1.1.0              |   py36h23d657b_0         144 KB
    mkl_random-1.1.1           |   py36h0573a6f_0         327 KB
    numpy-1.18.5               |   py36ha1c710e_0           5 KB
    numpy-base-1.18.5          |   py36hde5b4d6_0         4.1 MB
    six-1.15.0                 |             py_0          13 KB
    ------------------------------------------------------------
                                           Total:       135.4 MB

The following NEW packages will be INSTALLED:

  blas               pkgs/main/linux-64::blas-1.0-mkl
  h5py               pkgs/main/linux-64::h5py-2.10.0-py36hd6299e0_1
  hdf5               pkgs/main/linux-64::hdf5-1.10.6-hb1b8bf9_0
  intel-openmp       pkgs/main/linux-64::intel-openmp-2020.1-217
  libgfortran-ng     pkgs/main/linux-64::libgfortran-ng-7.3.0-hdf63c60_0
  mkl                pkgs/main/linux-64::mkl-2020.1-217
  mkl-service        pkgs/main/linux-64::mkl-service-2.3.0-py36he904b0f_0
  mkl_fft            pkgs/main/linux-64::mkl_fft-1.1.0-py36h23d657b_0
  mkl_random         pkgs/main/linux-64::mkl_random-1.1.1-py36h0573a6f_0
  numpy              pkgs/main/linux-64::numpy-1.18.5-py36ha1c710e_0
  numpy-base         pkgs/main/linux-64::numpy-base-1.18.5-py36hde5b4d6_0
  six                pkgs/main/noarch::six-1.15.0-py_0



Downloading and Extracting Packages
mkl-service-2.3.0    | 219 KB    | ############################################################################### | 100% 
numpy-base-1.18.5    | 4.1 MB    | ############################################################################### | 100% 
numpy-1.18.5         | 5 KB      | ############################################################################### | 100% 
six-1.15.0           | 13 KB     | ############################################################################### | 100% 
intel-openmp-2020.1  | 780 KB    | ############################################################################### | 100% 
h5py-2.10.0          | 901 KB    | ############################################################################### | 100% 
blas-1.0             | 6 KB      | ############################################################################### | 100% 
mkl-2020.1           | 129.0 MB  | ############################################################################### | 100% 
mkl_random-1.1.1     | 327 KB    | ############################################################################### | 100% 
mkl_fft-1.1.0        | 144 KB    | ############################################################################### | 100% 
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
```

so I think the answer is **yes** - you get hdf5 the shared library + h5py the python bindings

Reply:
That's nice and unlike pytables.

Reply:
I don't know pytables so I can't really respond.

I will add that, strangely, one must *also* specify `h5py` in the package requirements for things to work - otherwise you get an import error for `h5py`. This is odd to me, because I would have thought that `h5py` being installed by anaconda would make everything available and working regardless ... but it didn't.

Since `h5py` is a dependency of the project now I'm not bothered by that, but it maybe indicates that you *do not* get the python bindings (even though the anaconda `h5py` package is clearly the python package).